missing-dependency/missing-dependency.iml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <module org.jetbrains.idea.maven.project.MavenProjectsManager.isMavenModule="true" type="JAVA_MODULE" version="4">
-  <component name="NewModuleRootManager" LANGUAGE_LEVEL="JDK_1_8">
+  <component name="NewModuleRootManager" LANGUAGE_LEVEL="JDK_11">
```

Comment:
I made a change to this file, but I am not entirely sure what this module does. @motlin do we need this, it seems this was committed in 2018 with just this iml file. Happy to revert this change and take it up later.

Reply:
this should ideally auto-update. But the change is ok.